kubevirt_scripts/mountcd.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+#### IMPORTANT: This script is only meant to show how to implement required scripts to make custom hardware compatible with FakeFish. Dell hardware is supported by the `idrac-virtualmedia` provider in Metal3.
+#### This script has to mount the iso in the server's virtualmedia and return 0 if operation succeeded, 1 otherwise
+#### Note: Iso image to mount will be received as the first argument ($1)
+#### You will get the following vars as environment vars
+#### BMC_ENDPOINT - Has the BMC IP
+#### BMC_USERNAME - Has the username configured in the BMH/InstallConfig and that is used to access BMC_ENDPOINT
+#### BMC_PASSWORD - Has the password configured in the BMH/InstallConfig and that is used to access BMC_ENDPOINT
+
+ISO=${1}
+CLUSTER_STORAGE_CLASS=ocs-storagecluster-ceph-rbd
+IS_HTTPS=false
+
+if [ $(echo ${ISO} | grep -c https://) -eq 1 ]
+then
+  HOSTNAME=$(echo ${ISO} | awk -F "https://" '{print $2}' | awk -F "/" '{print $1}' | awk -F ":" '{print $1}')
+  PORT=$(echo ${ISO} | awk -F "https://" '{print $2}' | awk -F "/" '{print $1}' | awk -F ":" '{print $2}')
```

Comment:
nit: I'd rather have this implemented in Python with urllib.parse, to be honest.

Reply:
Agree. But the point is for these scripts to use bash only for now at least.